com.microsoft.mrtk.input/Utilities/PoseSource/FallbackCompositePoseSource.cs

```diff
@@ -54,7 +54,7 @@ public bool TryGetPose(out Pose pose)
         protected struct PoseSourceWrapper
         {
             [SerializeReference]
-            [InterfaceSelector]
+            [InterfaceSelector(true)]
```

Comment:
I can't think of a scenario where I'd have a Fallback pose source and want any of the fallbacks to be `null`. Am I missing a case?